app/src/main/res/layout/fragment_market_picker.xml

```diff
@@ -0,0 +1,20 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this enclosing `ConstraintLayout` necessary?

Reply:
Nope 😄